### PR TITLE
Update pl-PL.lua

### PR DESCRIPTION
**Pull request by gighul**
updated version, up to date
[pl_PL.txt](https://github.com/InZidiuZ/op-framework-public/files/5524329/pl_PL.txt)

### CODE REVIEW

**Comment by gighul** [comment]
[pl_PL.txt](https://github.com/InZidiuZ/op-framework-public/files/5524349/pl_PL.txt)
Sorry, this one. Previous one somehow did not updated itself to 2403 lines from 2387.

File: languages/pl-PL.lua [hunk 1]
```diff
@@ -47,6 +47,17 @@ OP.Global.Locales.Languages["pl-PL"] = {
...
+		spawned_vehicle_for_player = "Udało się zrespić `${modelName}` dla ${consoleName}.",	
+		spawned_vehicle_for_everyone = "Udało się zrespić `${modelName}` dla wszystkich.",	
+		spawn_vehicle_for_player_not_staff = "Gracz próbował zrespić pojazd dla kogoś, ale nie posiada odpowiednich uprawnień.",	
+		spawn_vehicle_for_self_not_staff = "Gracz próbował zrespić pojazd dla siebie, ale nie posiada odpowiednich uprawnień.",	
+		spawned_vehicle_for_self_title = "Zrespiono pojazd",	
+		spawned_vehicle_for_self_details = "${consoleName} zrespił pojazd o modelu i nazwie `${modelName}`.",	
+		spawned_vehicle_for_player_title = "Zrespiono pojazd dla gracza",	
+		spawned_vehicle_for_player_details = "${consoleName} zrespiono pojazd o modelu `${modelName}` dla gracza ${targetConsoleName}.",	
+		spawned_vehicle_for_everyone_title = "Zrespiono pojazd dla wszystkich",	
+		spawned_vehicle_for_everyone_details = "${consoleName} zrespiono pojazd o modelu `${modelName}` dla wszystkich.",
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:60]: At the end of all of these lines there is a random tab indent. Could this be resolved?

File: languages/pl-PL.lua [hunk 2]
```diff
@@ -325,6 +336,8 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		spawn_vehicle_command_help = "Respi pojazd.",
 		spawn_vehicle_command_parameter_model_name = "nazwa modelu",
 		spawn_vehicle_command_parameter_model_name_help = "Nazwa modelu pojazdu który próbujesz zrespić.",
+		spawn_vehicle_command_parameter_server_id = "serwer id",	
+		spawn_vehicle_command_parameter_server_id_help = "Serwer ID gracza dla którego chcesz zrespić pojazd. Możesz zostawić `0` by zrespić dla siebie.",
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:340]: Here too.

File: languages/pl-PL.lua [hunk 3]
```diff
@@ -943,14 +956,15 @@ OP.Global.Locales.Languages["pl-PL"] = {
...
-		bottom_part_2 = "State of San Andreas",
+		video = "Nagranie: ${video}",	
+		time = "Czas: ${time}",	
+		date = "Data: ${date}",	
+		unit_id = "ID jednostki: ${unitId}",	
+		unit_name = "Nazwa jednostki: ${unitName}",	
+		unit_speed_metric = "Prędkość jednostki: ${unitSpeed} km/h",	
+		unit_speed_imperial = "Prędkość jednostki: ${unitSpeed} mp/h",	
+		bottom_part_1 = "Ten pojazd należy do",	
+		bottom_part_2 = "State of San Andreas",
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:967]: On all of these as well... 🤔

File: languages/pl-PL.lua [hunk 4]
```diff
@@ -1002,12 +1016,14 @@ OP.Global.Locales.Languages["pl-PL"] = {
...
-		fuel = "PALIWO",
-		fps = "FPS",
+		mph = "mp/h",	
+		kmh = "km/h",	
+		ft = "ft",	
+		m = "m",	
+		belt = "Pasy",	
+		cruise = "Tempomat",	
+		fuel = "Paliwo",	
+		fps = "FPS",
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1026]: Seems to be that you have added tab indents at the end of every changed line. Not going to create more comments on the other ones, but please resolve these as to not pollute the file.

**Review by InZidiuZ — APPROVED** [review]

**Pull request merged**